src/components/consul/selectors/ImageSelector.js

```diff
@@ -171,7 +180,9 @@ export const ImageSelector = ({
           <Input {...item} control={control} hidden name={name} value={JSON.parse(value)} />
 
           <Button
-            disabled={values?.length >= 5}
+            disabled={
+              values?.length >= (parseInt(configuration?.limitation?.maxFileUploads?.value) ?? 5)
```

Comment:
we have defaults in the code, right. so the hardcoded `5` should come from that defaults i guess.

Reply:
Not until now. But I added `MAX_FILE_UPLOADS`.

Reply:
Or are you talking about the default data in `sueDefaultConfigurations`?